checker/build.xml

```diff
@@ -682,6 +682,8 @@
                printsummary="false"
                haltonerror="${halt.on.test.failure}"
                haltonfailure="${halt.on.test.failure}">
+            <!-- TODO: Figure out why we need javac.lib on bootclasspath when running test. -->
+            <jvmarg value="-Xbootclasspath/p:${javac.lib}"/>
```

Comment:
It looks like there are two failures:
- downstream projects: those need to be adapted separately, please do in another PR
- one expected error no longer occurs: This is rather mysterious. Can you remove that expected error and see if anything else fails? If nothing else fails, I would be fine with making that warning a TODO.